modules/dbfs/mysql_test.go

```diff
@@ -460,20 +386,52 @@ func TestDatabaseImpl_MySQLProjectRevokePermission(t *testing.T) {
 	}
 }
 
+func TestDatabaseImpl_MySqlUserProjectPermissionLookup(t *testing.T) {
+	configSetup(t)
+	di := new(DatabaseImpl)
+
+	di.MySQLUserDelete(userJoel.Username, "secret")
+	di.MySQLUserDelete(userAustin.Username, "secret")
+
+	defer func() {
```

Comment:
it's ensuring it's not in the database before the test starts, and ensuring it's deleting it once the test is done